### ISSUE TITLE
Update enable/disable setting descriptions

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#120559]
Brought up [through user feedback](https://twitter.com/_tomalak/status/1378989217905266688) that our settings that mention "enabled/disable" in the description are confusing:

<img width="600" src="https://user-images.githubusercontent.com/35271042/113591373-a948bd80-95e8-11eb-9f37-9a36b5753ec7.png">

I realize that this is inconsistent with how we describe other settings and tend to use "Enable" only, to make that clear:

<img width="450" src="https://user-images.githubusercontent.com/35271042/113591705-19574380-95e9-11eb-92a4-ecb92567b141.png">

**Comment by Tomalak** [comment]
[x] Show hover

is better than

[x] Controls whether hover is shown

From a UI metaphor point-of-view, checkboxes are not about an _action_, they are about a _state_.

The state is best described matter-of-factly for the "on" position. This rules out any description that focuses on the _transition_ of state, such as "Enable/disable" or "Enables". 

Since enabling/disabling is what the user does when clicking the box, "enable" or "disable" should be avoided in the label entirely. This also makes it easier to write documentation or instructions, as it avoids the _"please enable the 'Enable XYZ' checkbox"_ awkwardness.

More suitable verbs are direct actions: "Show XYZ", "Hide XYZ", "Validate XYZ", "Save before doing XYZ".

**Comment by misolori** [comment]
> From a UI metaphor point-of-view, checkboxes are not about an action, they are about a state.

This is a great point and makes me wonder how this can scale to the rest of our settings cc @Tyriar @roblourens

**Comment by Tomalak** [comment]
I can imagine where these descriptions come from - they are typical for requirements documents or user stories

> "...needs a way to enable/disable feature XYZ"

leads to a checkbox that says _"enable/disable feature XYZ"_.

But requirements documents are one abstraction layer *away* from the UI they describe. The wording from a requirements document is unsuitable for the UI labels - UI labels need to be the description of the feature, not the description of the description of the feature. :)

**Comment by misolori** [comment]
Here's a good example to see how this could scale and uses a mixture of `Show` and `Use`:

```jsonc
"html.format.enable.desc": "Use the default HTML formatter.",
"html.autoClosingTags": "Autoclose HTML tags.",
"html.mirrorCursorOnMatchingTag": "Mirror cursors on matching HTML tags.",
"typescript.validate.enable": "Show TypeScript validation.",
"typescript.format.enable": "Use the default TypeScript formatter.",
"javascript.validate.enable": "Show JavaScript validation.",
"javascript.referencesCodeLens.enabled": "Show references CodeLens in JavaScript files.",
"javascript.suggestionActions.enabled": "Show suggestion diagnostics for JavaScript files in the editor.",
"terminal.integrated.useWslProfiles": "Show WSL distros in the terminal dropdown",
```

**Comment by Tomalak** [comment]
Yep, all of that sounds *a lot* better.

**Comment by digitarald** [comment]
As we touch up the copy, we could consider to explain some of the key concepts (like _inline hints_ or _code lenses_) in the description. Most of the descriptions above just put words from the title in a different order.

**Comment by ntrogh** [comment]
@jo-oikawa Yep, let's do an audit, write down guidance, and do an edit pass to align settings descriptions.
